test/safe/push.test.ts

```diff
@@ -33,6 +38,12 @@ describe("Push Webhook", () => {
 	afterEach(async () => {
 		await Installation.destroy({ truncate: true });
 		await Subscription.destroy({ truncate: true });
+		//We have to restore Date.now to avoid errors in SQS Client.
+		//SQS Client uses Date.now and fails if it is "undefined"
+		//Hence we mock Date with jest, it will be returning undefined when we clean all mocks
+		//Some async operations might still be running (like message deletion) even after the test is finished.
+		//TODO Instead stop the queue every time after each test. However it would be possible only when we get rid of Redis because bull queues are not restartable
+		global.Date.now = realDateNow;
```

Comment:
That's something we have to deal with now until we get rid of Redis queue and will be able to restart worker in beforeEach / afterEach. (Redis queue doesn't allow restart)

Currently for end-to-end test we start worker in beforeAll and stop it in afterAll.

Before tests when jest mocks are reset, Date.now() can return "undefined" which can cause errors in currently running SQS listener